src/jit/compiler.h

```diff
@@ -6931,6 +6929,17 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
     // Should we support SIMD intrinsics?
     bool featureSIMD;
 
+    // Have we identified any SIMD types?
+    bool _usesSIMDTypes;
+    bool usesSIMDTypes()
```

Comment:
I don't think it needs to be a PhasedVar, though it's probably the case that all the places we set it to true (in the importer) are before the use (in `fgPromoteStructs()`). It's actually just an optimization to avoid getting information about a struct-type field if we haven't seen any SIMD types.